### ISSUE TITLE
.

### ISSUE THREAD

**Issue by v1rtl** [ruslanchek/vscode-roadrunner#1]
.

**Comment by m-brtn** [comment]
I'm sorry, but I'm not familiar with Windows and it's WSL. But I think `/mnt/c` is a correct way to access resource `c:/` on Windows through WSL. Could you bring some more info about your problem, please? Extension can't access your workspace files /mnt/c/.../package.json?

**Comment by talentlessguy** [comment]
@ruslanchek, yes, it starts a task in an integrated terminal, which is, in my case Fish, but doesn't change its path. Would be nice if the plugin could detect if wsl is installed. To check whether WSL is installed, you could use a cmd command:

```bat
which wsl.exe
```

or check if bash is accessible:

```
which bash
```

**Comment by alyxpele** [comment]
Alternatively, it doesn't work with msys2/mingw terminal integration:
`cd c:\msys64\home\[...] && npm run watch`
It should be instead:
`cd /c/msys64/home/[...] && npm run watch`

Edit: Windows paths are supported under MinGW, but backslash should be escaped:
`cd c:\\msys64\\home\\[...] && npm run watch` will work too
